code-challenges.test.js

```diff
@@ -11,41 +11,110 @@
+//         expect(multipliedByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+//         expect(multipliedByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+//     })
+// })
+
+
+// // const numbersArray1 = [6, 7, 8, 9, 10]
+// // // Expected output: [18, 21, 24, 27, 30]
+// // const numbersArray2 = [24, 27, 30, 33, 36]
+// // // Expected output: [72, 81, 90, 99, 108]
+
+// // b) Create the function that makes the test pass.
+
+// // Pseudo code:
+// //input:
+// //output:
+// //iterate through the array using .map
+// //each interaction multiplies the current value by three
+// //the product is returned in the new array made
+
+// const multipliedByThree = (arr) => {
+//     return arr.map(num => {
+//         return num * 3
+//     })
+// }
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
-// b) Create the function that makes the test pass.
-
-// Pseudo code:
+// // a) Create a test with expect statements for each of the variables provided.
+// describe('divisibleByThree',() => {
+//     it('takes a object as an argument and decides if the number inside it is evenly divisible by three or not',() => {
+//         const object1 = { number: 15 }
+//         // Expected output: "15 is divisible by three"
+//         const object2 = { number: 0 }
+//         // Expected output: "0 is divisible by three"
+//         const object3 = { number: -7 }
+//         // Expected output: "-7 is not divisible by three"
+//         expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+//         expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+//         expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+//     })
+// })
+
+// // const object1 = { number: 15 }
+// // // Expected output: "15 is divisible by three"
+// // const object2 = { number: 0 }
+// // // Expected output: "0 is divisible by three"
+// // const object3 = { number: -7 }
+// // // Expected output: "-7 is not divisible by three"
+
+// // b) Create the function that makes the test pass.
+
+// // Pseudo code:
+// //Input: An object with numbers
+// //Output: An object that decides if the numbers inside are divisible by three
+// //using % iterate through each object's number
+// //using string interpelation, each product returns a string describing the outcome
+
+// const divisibleByThree = (string) => {
+//     let num1 = object1
+//     let num2 = object2
+//     let num3 = object3
+//     if(num1 % 3 === 0) {
+//         retrun `${num1} is divisible by three`
```

Comment:
You are trying to hard code the test variables as separate variables in your code. Remember that you are performing a function call in your test, which will pass the test variables as arguments into your function. Arguments then take the place of the parameter. In this case, `string` is your parameter.

Think about what is being passed into the function. For this challenge, an object is given to your function. In your code, you are trying to perform math on an object, which will throw an error. How do you access the number inside the object? Hint: we use dot notation to access key value pairs in objects.